testing/tests/audio.lua

```diff
@@ -0,0 +1,296 @@
+-- love.audio
+
+
+-- love.audio.getActiveEffects
+love.test.audio.getActiveEffects = function(test)
+  -- tests
+  test:assertNotEquals(nil, love.audio.getActiveEffects(), 'check not nil')
+  test:assertEquals(0, #love.audio.getActiveEffects(), 'check no effects running')
+  love.audio.setEffect('testeffect', {
+    type = 'chorus',
+    volume = 10
+  })
+  test:assertEquals(1, #love.audio.getActiveEffects(), 'check 1 effect running')
+  test:assertEquals('testeffect', love.audio.getActiveEffects()[1], 'check effect details')
+end
+
+
+-- love.audio.getActiveSourceCount
+love.test.audio.getActiveSourceCount = function(test)
+  -- tests
+  test:assertNotEquals(nil, love.audio.getActiveSourceCount(), 'check not nil')
+  test:assertEquals(0, love.audio.getActiveSourceCount(), 'check 0 by default')
+  local testsource = love.audio.newSource('resources/click.ogg', 'static')
+  test:assertEquals(0, love.audio.getActiveSourceCount(), 'check not active')
+  love.audio.play(testsource)
+  test:assertEquals(1, love.audio.getActiveSourceCount(), 'check now active')
+  love.audio.pause()
+  testsource:release()
```

Comment:
It'd be nice to avoid having to use `object:release()` in every test that uses an object. If you `collectgarbage("collect")` between every test does it significantly increase the time it takes for all tests to run? If not I think I'd prefer something like that instead of having release() inside tests.